src/topology/config/mod.rs

```diff
@@ -332,6 +335,35 @@ impl Config {
         self.global.dns_servers.sort();
         self.global.dns_servers.dedup();
 
+        // If the user has multiple config files, we must *merge* log schemas until we meet a
+        // conflict, then we are allowed to error.
+        let default_schema = event::LogSchema::default();
```

Comment:
nit: prob could just do `if with.global.log_schema = LogSchema::default()`

Reply:
We use it later though :o